### DISCUSSION TITLE
dbt Power User doesn't seem to work for versions of dbt-postgres higher than v1.4.1

### DISCUSSION THREAD

**Discussion by james-mead** [AltimateAI/vscode-dbt-power-user#788] [General]
I've been an active user of dbt Power User for at least a couple of years now, and I moved to a new role at the start of this year which makes use of a Postgres database as the data warehouse, however I've found that any attempt to install dbt-postgres locally on versions higher than 1.4.1 results in Power User losing all functionality and all kinds of error messages pop up.

I was just wondering whether this is a known issue or whether I'm the only one experiencing this?

**Comment by anandgupta42** [comment]
@james-mead This is the first time someone has brought this to our notice. Can you please file a bug with instructions for reproducing the problem?
